### DISCUSSION TITLE
Reconstructed waveform from icwt is still different from the original timeseries.

### DISCUSSION THREAD

**Discussion by beta-yang** [regeirk/pycwt#47] [Q&A]
After fixing the bug from PR( #29 ), I test the following code and plot the original timeseries and iwave, but they are not fully same.

W, scales, freqs, coi, _, _ = wavelet.cwt(s, dt, dj, s0, J, mother)
iwave = wavelet.icwt(W, scales, dt, dj, mother)

And it seems in the example of https://pycwt.readthedocs.io/en/latest/, there is the same problem in fig a).
Have anyone met this and how to solve it?
Thanks a lot!

**Comment by Malley1** [comment]
Hi, I'm having a similar problem. See attached plots: 1) blue = input signal, orange = reconstructed. 2) Difference i.e. orange minus blue. Would be grateful for any suggestions
![diff_icwt1](https://user-images.githubusercontent.com/26087210/165788037-63740404-8584-417f-8490-d29095be58d3.png)
![diff_icwt2](https://user-images.githubusercontent.com/26087210/165788058-eb8e4df3-8a43-484d-aa68-3f9e65c4c627.png)
!
